Hudl.FFprobe/Serialization/Converters/FrameConverter.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using Hudl.FFmpeg.Metadata.FFprobe.BaseTypes;
+using Hudl.FFprobe.Metadata.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+
+namespace Hudl.FFprobe.Serialization.Converters
+{
+    internal class FrameConverter : CustomCreationConverter<List<BaseFrameMetadata>>
+    {
+        public override List<BaseFrameMetadata> Create(Type objectType)
+        {
+            throw new NotImplementedException();
```

Comment:
Is this just not needed? If it's waiting for something else to be implemented first or something, you may want to add a message to the exception.